File Templates/Custom Templates/Reactor Controller.xctemplate/CollectionVIewController/___VARIABLE_ID___LayoutComposer.swift

```diff
@@ -0,0 +1,90 @@
+//___FILEHEADER___
```

Comment:
Tento subor sa vola CollectionVIewController to slovo View by malo mat male pismenko i

Prebehni aj zvysne subory bo aj tam je podobna chyba